client/src/serialize/txt/rdata_parsers/tlsa.rs

```diff
@@ -8,12 +9,17 @@
 
 //! tlsa records for storing TLS authentication records
 
-use data_encoding::hex;
-
 use error::*;
 use rr::rdata::TLSA;
 use rr::rdata::tlsa::CertUsage;
 
+const HEX: ::data_encoding::Encoding = new_encoding!{
+    symbols: "0123456789abcdef",
+    ignore: " \t\r\n",
+    translate_from: "ABCDEF",
```

Comment:
I didn't look at this in enough detail before... on line 67 I have a `to_upper()` b/c in DNS it's stored as lower... is this no longer necessary?

Reply:
Yes you're right. I missed that. It is no longer necessary.